app/javascript/dashboard/components/widgets/conversation/bubble/integrations/Dyte.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div>
+    <woot-button
+      size="small"
+      variant="smooth"
+      color-scheme="secondary"
+      icon="video-add"
+      class="join-call-button"
+      :is-loading="isLoading"
+      @click="joinTheCall"
+    >
+      {{ $t('INTEGRATION_SETTINGS.DYTE.CLICK_HERE_TO_JOIN') }}
+    </woot-button>
+    <div v-if="dyteAuthToken" class="video-call--container">
+      <iframe
+        :src="meetingLink"
+        allow="camera;microphone;fullscreen;display-capture;picture-in-picture;clipboard-write;"
+      />
+      <woot-button
+        size="small"
+        variant="smooth"
+        color-scheme="secondary"
+        class="join-call-button"
+        @click="leaveTheRoom"
+      >
+        {{ $t('INTEGRATION_SETTINGS.DYTE.LEAVE_THE_ROOM') }}
+      </woot-button>
+    </div>
+  </div>
+</template>
+<script>
+import DyteAPI from 'dashboard/api/integrations/dyte';
+
+const DYTE_MEETING_LINK = 'https://app.dyte.in/meeting/stage/';
```

Comment:
Should we have this in an `env`, with a fallback to the `app.dyte.in` domain? 
It makes sense to do this only if it is a possibility that dyte can be hosted on a separate domain? Also given their website is `dyte.io` and API is `dyte.in` there is a possibility the domains may change in the future

Reply:
I don’t think so.

Reply:
I have not found the self-hosted option in their docs. If it is there, I can pick it up as an enhancement.